lang/jruby/Portfile

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+name                jruby
+version             9.2.17.0
+revision            0
+java.version        1.8+
```

Comment:
Please add an appropriate` java.fallback` here to indicate the java version to install if the requirement is not satisfied.